Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/CommonConst.java

```diff
@@ -53,6 +53,7 @@
     public static final String ToolWindowSparkJobDebugIcon_13x_Path = "/icons/ToolWindowSparkJobDebugger_13x.png";
     public static final String ToolWindowSparkJobDebugIcon_16x_Path = "/icons/ToolWindowSparkJobDebug.png";
 
+    public static final String ENABLE_HDINSIGHT_NEW_SDK = "false";
```

Comment:
It can be `boolean` type

Reply:
This is my mistake, `ENABLE_HDINSIGHT_NEW_SDK` should be the property name, which should be something like `Enable.HDInsight.New.SDK`.

---
In reply to: [271120721](https://github.com/Microsoft/azure-tools-for-java/pull/2964#discussion_r271120721) [](ancestors = 271120721)